nlp_primitives/utils.py

```diff
@@ -111,3 +115,19 @@ def valid_dfs(es, aggregations, transforms, feature_substrings,
     if not multi_output:
         assert len(applicable_features) == df.shape[1]
     return
+
+
+def unpack_data():
+    if not os.path.exists(nltk.data.path[0]):
+        fp = os.path.normpath(os.path.join(os.path.realpath(__file__), '../../data/nltk-data.tar.gz'))
```

Comment:
Can we use `pathlib` instead of `os.path`? Its much easier to use :)

https://docs.python.org/3/library/pathlib.html